GitUI/CommandsDialogs/FormCleanupRepository.cs

```diff
@@ -110,21 +110,16 @@ private void checkBoxPathFilter_CheckedChanged(object sender, EventArgs e)
 
         private void AddPath_Click(object sender, EventArgs e)
         {
-            var dialog = new CommonOpenFileDialog
+            var dialog = new FolderBrowserDialog()
```

Comment:
❔ Are we losing any functionality by this change?

Reply:
No, the original code was creating the modern folder browser dialog, which Windows Forms received in .NET Core timeframe only (previously it was old pre-Vista style). See https://github.com/dotnet/winforms/pull/57
Replacing with the Windows Forms variant now.